### DISCUSSION TITLE
Manifoldapp documents in the wider environment - DOIs, export of text as pdf, reference managers

### DISCUSSION THREAD

**Discussion by medical-librarian** [ManifoldScholar/manifold#3571] [Ideas]
Hello, I've browsed a bit in the 'Journal of Interactive Pedagogy and Technology' as we are currently looking into platforms we might use for publishing both research output and OER, and are considering Manifoldapp. A few issues I noticed>

- There were no DOIs - can Manifold handle DOIs?
- It was also not possible to export text as pdf which is still useful for annotating and working with a text. 
- Zotero couldn't import articles in a meaningful way, either - incorrect metadata, no pdf. 
So, for a scholarly workflow, the journal, as it stands, is not very convenient - although these articles are mere text and in no way interactive, strewn with multimedia, or whatever. Is this due to limitations of the software? And if so, are there plans (and a time frame) for remedying these shortcomings? Thanks and kind regards, Gerhard

**Comment by tsmyre** [comment]
Hello, and thank you for your interest in Manifold. And apologies for our delay in responding. We have some thoughts below tied to each of your bullet points. To help facilitate this conversation I’m going to move this over into our discussion section. We look forward to your thoughts. And thanks again!

- Manifold can handle DOIs, but it cannot mint them; that is up to the publisher to do outside of the system.
- Manifold does not export to PDF by design, however you can print to PDF from the Manifold reader in a way that is visually pleasing.
- We can’t speak to how the Zotero app works, but the presentation and dissemination of metadata is a conversation we are actively having. Right now the citation tool we have automatically creates references based on the metadata saved in the system. We are exploring adding both more fields as well a more human component to that that would likely interface better with reference managers.

**Comment by amandafrench** [comment]
@tsmyre Just curious: what do you mean that Manifold can "handle" DOIs? Does that mean it has a field to store a DOI minted elsewhere for a particular publication?

**Reply by tsmyre** [reply]
@amandafrench Yes, that is exactly right. Manifold has a field for DOIs at the project, text, resource, and resource collection level for DOIs that are minted elsewhere.
